docs/_data/prague-2022-config.yaml

```diff
@@ -0,0 +1,197 @@
+    #   link: /welcome-wagon
+    # - text: See the Schedule!
+    #   link: /schedule
+    # - text: Buy a Ticket!
+    #   link: /tickets
+    - text: Sponsor us
+      link: /sponsors/prospectus
+    #   - text: Submit a Talk
+    #     link: /cfp
+    #- text: See the talks!
+    #  link: /speakers
+    #- text: See the Schedule!
+    #  link: /schedule
+    # - text: Watch the Live Stream
+    #   link: /livestream
+    # - text: See the sketchnotes
+    #   link_absolute: https://www.flickr.com/photos/writethedocs/albums/72157720128594755
+    # - text: Watch the videos!
+    #   link_absolute: https://www.youtube.com/playlist?list=PLZAeFn6dfHpnaoiOQyd9BYbQbprDGQjQ9
+  bottom:
+    - text: Welcome to our conference
+      link: /news/welcome
+    # - text: See the Schedule!
+    #   link: /schedule
+#     - text: Date and ticket information
+#       link: /news/announcing-virtual-conf-dates-tickets
+    - text: Sponsor the conference
+      link: /sponsors/prospectus
+    # - text: See the talks!
+    #   link: /speakers
+#     - text: Buy a Ticket!
+#       link: /tickets
+    # - text: Prepare with the Welcome Wagon
+    #   link: /welcome-wagon
+    # - text: See the schedule!
+    #   link: /schedule
+    # - text: Watch the Live Stream
+    #   link: /livestream
+    # - text: Browse the photos
+    #   link_absolute: https://www.flickr.com/photos/writethedocs/albums/72157691507514803
+    # - text: Watch the videos!
+    #   link_absolute: https://www.youtube.com/playlist?list=PLZAeFn6dfHpkBJAPYFrob6gqdiBuePwGJ
+
+tickets:
+  corporate:
+    price: €150
+  independent:
+    price: €100
+  student:
+    price: €25
+
+sponsorship:
+  second_draft:
+    price: €1,500
+  publisher:
+    price: €3,000
+  patron:
+    price: €6,500
+  keystone:
+    price: €12,500
+
+
+date: # how do we handle these? human readable would be nice.
+  main: "**September 11-13, 2022, online**"
+  short: Sep 11-13, 2023
+  tickets_live: "June 2022"
+  month: September
+  # Conference is special for the index to show the combined conference days
+  conference:
+    event: Main Conference
+    date: September 12-13
+    summary: The main days of the conference. We will be running one main track, and an <a href="/conf/prague/2022/unconference/">Unconference</a>.
+    icon: conference
+  # These are listed in the schedule
+  day_one:
+    event: Hike
+    date: On your own
+    summary: We encourage you to go on a hike in your own beautiful home town.
+    #summary: If you're in town early, join us for the Hike. prague has excellent access to the woods right in the city, and we'll be exploring Forest Park and the Hoyt Arboretum.
+    icon: hike
+    dotw: Saturday
+  day_two:
+    event: Writing Day
+    date: September 11
+    summary: Join us for the <a href="/conf/prague/2022/writing-day/">Writing Day</a> and Welcome Reception. The first official day of the conference is full of chances to interact with other documentarians.
```

Comment:
This URL will be broken until the full site is up, is there a way to hide it rather than delete it? Or should we just delete it?

Reply:
I think the URL will work - the pages are there, just unlisted in the nav. I did look at the page and did not see a problem with the current content.